### DISCUSSION TITLE
Using remote schema?

### DISCUSSION THREAD

**Discussion by nbbaier** [ardatan/feTS#411] [Ideas]
First, love what this library is doing! It's really cool and I'm excited to explore it more. One thing that I wonder is it would be possible to enable remote schema. That is, having some way of fetching and parsing a schema directly from a URL instead of having to have it locally to import?

**Comment by ardatan** [comment]
It is not possible due to the limitations in TypeScript. So you have to download the schema on your own and save it. 
https://github.com/microsoft/TypeScript/issues/35749

**Reply by nbbaier** [reply]
Got it thanks!
